source/mapping-rules/fields/data-type-conversion-guide.txt

```diff
@@ -0,0 +1,229 @@
+.. _rm-data-type-conversion-guide:
+
+==========================
+Data Type Conversion Guide
+==========================
```

Comment:
(info typing suggestion) 
What about updating the title to Data Type Conversion Reference?
My reasoning is:
- This is a reference page, where you can expect to get very detailed information. We might just call it that, straightforwardly.
- At MDB we tend to use "guide" to imply a set of tasks. See our Guides page for examples: https://www.mongodb.com/docs/guides/